<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalculator</class>
 <widget class="QMainWindow" name="DepositCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>721</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>705</width>
    <height>721</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon.png</normaloff>../assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,1" columnstretch="0,1,0">
    <item row="3" column="2">
     <widget class="QComboBox" name="rate_type_input">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Per month</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Per annum</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="term_type_input">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Days</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Years</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="term_label">
      <property name="text">
       <string>Term</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QPushButton" name="calculate_button">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="term_input">
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QDoubleSpinBox" name="sum_input">
      <property name="showGroupSeparator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="suffix">
       <string> RUB</string>
      </property>
      <property name="minimum">
       <double>100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>500.000000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="rate_capitalization_input">
      <property name="text">
       <string>Rate capitalization</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QDateEdit" name="term_start_input">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1999</year>
        <month>12</month>
        <day>29</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="rate_label">
      <property name="text">
       <string>Rate</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
        <item>
         <widget class="QLabel" name="withdrawals_list_label">
          <property name="text">
           <string>Withdrawals list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="show_withdrawals_list">
          <property name="text">
           <string>Show (0)</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="minimum_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Minimum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="minimum_input">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> RUB</string>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="sum_label">
      <property name="text">
       <string>Sum</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QTableWidget" name="result_table">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Accrued interest</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Replenishment</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rest</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="rate_input">
      <property name="suffix">
       <string> %</string>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <double>5.700000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="payment_periodicity_label">
      <property name="text">
       <string>Payment periodicity</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QScrollArea" name="withdrawals_list">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="withdrawals_list_layout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>435</width>
         <height>98</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="withdrawals_widgets"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="add_withdrawal_button">
            <property name="text">
             <string>Add withdrawal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_withdrawals_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QScrollArea" name="refills_list">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="refills_list_layout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>435</width>
         <height>98</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="refills_widgets"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="add_refill_button">
            <property name="text">
             <string>Add refill</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_refills_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="payment_periodicity_input">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>Daily</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Weekly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Monthly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once in quarter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once in half of year</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Once in year</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>At the end of the term</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="term_start_label">
      <property name="text">
       <string>Term start</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
      <item>
       <widget class="QLabel" name="refills_list_label">
        <property name="text">
         <string>Refills list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_refills_list">
        <property name="text">
         <string>Show (0)</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionMath_and_plots_calculator"/>
    <addaction name="actionCredit_calculator"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <action name="actionMath_and_plots_calculator">
   <property name="text">
    <string>Math and plots calculator</string>
   </property>
  </action>
  <action name="actionCredit_calculator">
   <property name="text">
    <string>Credit calculator</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sum_input</tabstop>
  <tabstop>term_input</tabstop>
  <tabstop>term_type_input</tabstop>
  <tabstop>term_start_input</tabstop>
  <tabstop>rate_input</tabstop>
  <tabstop>rate_type_input</tabstop>
  <tabstop>rate_capitalization_input</tabstop>
  <tabstop>payment_periodicity_input</tabstop>
  <tabstop>refills_list</tabstop>
  <tabstop>add_refill_button</tabstop>
  <tabstop>clear_refills_button</tabstop>
  <tabstop>withdrawals_list</tabstop>
  <tabstop>add_withdrawal_button</tabstop>
  <tabstop>clear_withdrawals_button</tabstop>
  <tabstop>minimum_input</tabstop>
  <tabstop>calculate_button</tabstop>
  <tabstop>result_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
